src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryUntypedObject.cs

```diff
@@ -297,37 +297,71 @@ public static FormulaValue Float_UO(EvalVisitor runner, EvalVisitorContext conte
             return GetTypeMismatchError(irContext, BuiltinFunctionsCore.Value_UO.Name, DType.Number.GetKindString(), impl);
         }
 
-        public static FormulaValue Text_UO(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, UntypedObjectValue[] args)
-        {
-            var impl = args[0].Impl;
-
+        public static FormulaValue Text_UO(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, FormulaValue[] args)
+        {
+            var arg0 = (UntypedObjectValue)args[0];
+            var impl = arg0.Impl;
+

```

Comment:
What happens with the third arg to Text?

```
> Text(Value("1,234","fr-fr"),"","fr-fr")
"1,234"

> Text(Value("1,234"),"","fr-fr")
"1234"
```

Reply:
I removed support for the 3rd argument.